vtr_flow/scripts/run_vtr_flow.py

```diff
@@ -322,6 +322,13 @@ def vtr_command_argparser(prog=None):
         dest="odin_config",
         help="Supplies Odin with a custom config file for optimizations.",
     )
+    odin.add_argument(
```

Comment:
I suggest a more fullsome description.  Are these files pasted into the main benchmark circuit file?  (i.e. define what include means in this context).